README.md

```diff
@@ -1,7 +1,7 @@
 # Calendar
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_calendar/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_calendar/report/html_report/)
+- [DEMO LINK](https:TenetaJ.github.io/layout_calendar/)
```

Comment:
```suggestion
- [DEMO LINK](https://TenetaJ.github.io/layout_calendar/)
```
the same below

Reply:
added two slashes in file readme.md, but I don't know how to add them to the pull request at this stage

Reply:
P.S. please check this task: https://github.com/TenetaJ/layout_catalog